index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="css/my_portfolio.css">
+</head>
+<body>
+    <header id="header" class="header"> 
+        <div class="logo">John Doe</div>
+        <div class="burger"><img src="assets/images/burger.png" alt="" class="d-none-lg"></div>
```

Comment:
- [ ] Your Figma design looks good. you've done a lot of nice work on it 👍. However as per the Figma requirements It is required that the hamburger button have a `nav` tag. please implement this change it will make your HTML more semantically correct as a `nav` tag implies that there is some navigation functionality from the hamburger Thanks you.